theseus/labs/lie_functional/__init__.py

```diff
@@ -2,3 +2,56 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
+from typing import Callable, Optional, Sequence
+
+import torch
+
+import theseus.labs.lie_functional.se3 as _se3_impl
+import theseus.labs.lie_functional.so3 as _so3_impl
+from .constants import DeviceType
+from .lie_group import BinaryOperatorFactory, UnaryOperatorFactory
```

Comment:
Any reason for explicit path first two and relative path for last two?

Reply:
I think this wasn't working for some reason.

Reply:
Oh strange.

Reply:
Fixed in another PR.